src/components/FindProjectTutorial.vue

```diff
@@ -0,0 +1,321 @@
+
+      const icon = instructions.icon
+
+      return {
+        type: 'info' as const,
+        title: instructions.title,
+        text: instructions.description,
+        icon: icon ? matchIcon(icon) : undefined,
+      }
+    },
+  },
+  methods: {
+    fetchTutorialGroups() {
+      if (this.tutorial?.projectId) {
+        onValue(
+          // FIXME: verify group id
+          getGroupsRef(this.tutorial.projectId),
+          (snapshot) => {
+            const data = snapshot.val()
+            const groupKeys = Object.keys(data)
+            this.fetchTutorialProject(groupKeys[0])
+          },
+          (error) => {
+            console.error('Error fetching tasks for the tutorial', error)
+          },
+          { onlyOnce: true },
+        )
+      }
+    },
+    fetchTutorialProject(groupId: string | undefined) {
+      if (this.tutorial?.projectId && groupId) {
+        onValue(
+          getTasksRef(this.tutorial.projectId, groupId),
+          (snapshot) => {
+            const data = snapshot.val()
+            this.tasks = decompressTasks(data)
+            this.results = this.tasks.reduce(
+              (acc, val) => {
+                // Fill all the result with initial value
+                acc[val.taskId] = this.options[0].value
+
+                return acc
+              },
+              {} as Record<string, number>,
+            )
+          },
+          (error) => {
+            console.error('Error fetching tasks for the tutorial', error)
+          },
+          { onlyOnce: true },
+        )
+      }
+    },
+    nextTask() {
+      if (!this.isLastTask) {
+        if (this.isLastTask) {
+          return
+        }
+
+        this.selectedTasks = {}
+        this.currentTaskIndex += 1
+        this.userAttempts = 0
+        this.answersRevealed = false
+      }
+    },
+    updateResult(taskId: string) {
+      if (!this.answersRevealed) {
+        this.userAttempts += 1
+        const currentResult = this.results[taskId]
+
+        if (!isDefined(currentResult)) {
+          this.results[taskId] = this.options[0].value
+          return
+        }
+
+        const newValue = this.optionMap[this.optionMap[currentResult].nextOptionKey].value
+
+        this.results[taskId] = newValue
+      }
+    },
+    handleTileClicked(taskId: string) {
+      this.updateResult(taskId)
+    },
+    handleTileSelected(newValue: boolean, taskId: string) {
+      this.selectedTasks[taskId] = newValue
+    },
+    showAnswer() {
+      this.answersRevealed = true
+      this.taskList.forEach((task) => {
+        this.results[task.taskId] = task.referenceAnswer
+      })
+    },
+  },
+  emits: ['tutorialComplete'],
+  mounted() {
+    goOnline(db)
+    this.fetchTutorialGroups()
+    // this.fetchTutorialProject();
+  },
+})
+</script>
+
+<template>
+  <v-container>
+    <v-row>
+      <v-col>
+        <div>
+          {{ instructionMessage }}
+        </div>
+      </v-col>
+    </v-row>
+    <v-row v-if="alertContent">
```

Comment:
In the mobile app, these messages are placed over the image grid. This would reduce the vertical space and the need for scrolling while working through the tutorial.

Reply:
Actions has been moved inside alert to reduce vertical space!

Reply:
This is a great improvement, looks good to me!